docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md

```diff
@@ -0,0 +1,509 @@
+---
+layout: default
+title: Integrations User Guide
+pagename: integrations-user-guide
+lang: en
+---
+
+<!-- omit in toc -->
+# Integrations user guide
+
+In this guide:
+- [Introduction](#introduction)
+- [Getting started with {% include product %} Desktop](#getting-started-with--include-product--desktop)
```

Comment:
This link isn't working, maybe try it without the extra dashes?

Reply:
I think it's the include variable inside the link, I'll see if I can replace that with the variable

Reply:
I've had to remove the product name variable completely from the title in order to fix this issue.